src/kili/use_cases/api_key/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""Api Keys use cases."""
+import warnings
+from datetime import datetime, timedelta
+from typing import Generator, Optional
+
+from kili.adapters.kili_api_gateway import KiliAPIGateway
+from kili.adapters.kili_api_gateway.helpers.queries import QueryOptions
+from kili.domain.api_key import ApiKeyFilters
+from kili.domain.types import ListOrTuple
+
+
+class ApiKeyUseCases:
+    """Api Keys use cases."""
+
+    def __init__(self, kili_api_gateway: KiliAPIGateway) -> None:
+        """Init AssetUseCases."""
+        self._kili_api_gateway = kili_api_gateway
```

Comment:
I don' t remember if we have a baseusecases class for this?

Reply:
no we don't, I add one